contributing-device-support.md

```diff
@@ -1,19 +1,25 @@
-# Custom Board Support
+# Customer Board Support
+
+There are several options for onboarding the hardware onto balenaOS:
+
+1. You can find another board that would work for your case that is supported in our [supported device list](https://www.balena.io/docs/reference/hardware/devices/).
+2. You can consider our [Custom Device Support (CDS)](https://www.balena.io/pricing/#custom-device-support), which is a paid service during which we onboard the device for you and maintain it for a monthly fee. [Here](https://drive.google.com/file/d/17-DTeXkflkzW8Qduj48ut7JNUTWqiRv-/view?usp=sharing) is an overview including pricing.
```

Comment:
```suggestion
2. Consider our [Custom Device Support](https://www.balena.io/pricing/#custom-device-support) (CDS) service. This is a paid service where we onboard the device to balenaOS for you and maintain it for a monthly fee. Checkout out the [pricing for CDS](LINK) and how to get the process started for you. 
```

`Here` is not an accessible word for hyperlinks for folks reading our documentation using Read Aloud software. Added a suggestion for the sentence. 
Drive links are private and can easily deprecate. Please add a public link instead as we talked about.